diff --git a/run.sh b/run.sh
index ac53da5..8163514 100755
--- a/run.sh
+++ b/run.sh
@@ -106,30 +106,30 @@ PID_LIST+=($SUDO_UPF_PID $UPF_PID)
 
 sleep 1
 
-NF_LIST="nrf amf smf udr pcf udm nssf ausf"
-
-export GIN_MODE=release
-
-for NF in ${NF_LIST}; do
-    ./bin/${NF} -c ./config/${NF}cfg.yaml -l ${LOG_PATH}${NF}.log -lc ${LOG_PATH}${LOG_NAME} &
-    PID_LIST+=($!)
-    sleep 0.1
-done
-
-if [ $N3IWF_ENABLE -ne 0 ]; then
-    N3IWF_IKE_BIND_ADDRESS="127.0.0.21"
-    N3IWF_UE_ADDR="10.0.0.1/24"
-    sudo ip link add name ipsec0 type vti local ${N3IWF_IKE_BIND_ADDRESS} remote 0.0.0.0 key 5
-    sudo ip addr add ${N3IWF_UE_ADDR} dev ipsec0
-    sudo ip link set ipsec0 up
-    sleep 1
-
-    sudo ./bin/n3iwf -c ./config/n3iwfcfg.yaml -l ${LOG_PATH}n3iwf.log -lc ${LOG_PATH}${LOG_NAME} &
-    SUDO_N3IWF_PID=$!
-    sleep 1
-    N3IWF_PID=$(pgrep -P $SUDO_N3IWF_PID)
-    PID_LIST+=($SUDO_N3IWF_PID $N3IWF_PID)
-fi
+# NF_LIST="nrf amf smf udr pcf udm nssf ausf"
+
+#export GIN_MODE=release
+#
+#for NF in ${NF_LIST}; do
+#    ./bin/${NF} -c ./config/${NF}cfg.yaml -l ${LOG_PATH}${NF}.log -lc ${LOG_PATH}${LOG_NAME} &
+#    PID_LIST+=($!)
+#    sleep 0.1
+#done
+#
+#if [ $N3IWF_ENABLE -ne 0 ]; then
+#    N3IWF_IKE_BIND_ADDRESS="127.0.0.21"
+#    N3IWF_UE_ADDR="10.0.0.1/24"
+#    sudo ip link add name ipsec0 type vti local ${N3IWF_IKE_BIND_ADDRESS} remote 0.0.0.0 key 5
+#    sudo ip addr add ${N3IWF_UE_ADDR} dev ipsec0
+#    sudo ip link set ipsec0 up
+#    sleep 1
+#
+#    sudo ./bin/n3iwf -c ./config/n3iwfcfg.yaml -l ${LOG_PATH}n3iwf.log -lc ${LOG_PATH}${LOG_NAME} &
+#    SUDO_N3IWF_PID=$!
+#    sleep 1
+#    N3IWF_PID=$(pgrep -P $SUDO_N3IWF_PID)
+#    PID_LIST+=($SUDO_N3IWF_PID $N3IWF_PID)
+#fi
 
 wait ${PID_LIST}
 exit 0
